simulator/setting_up_simulator.md

```diff
@@ -6,27 +6,41 @@ title: Setting up the Simulator
 Setting up the Simulator
 ========================
 
-## Prerequisites
+# Prerequisites
```

Comment:
:x: this puts this heading at the same level (`<h1>`) as the "Setting up the Simulator" heading above, which is confusing and technically invalid.

Reply:
1b1c6a7